app/services/hotkeys.ts

```diff
@@ -1,31 +1,44 @@
+import { defer } from 'lodash';
 import { StreamingService } from 'services/streaming';
 import { ScenesService } from 'services/scenes';
 import { SourcesService } from 'services/sources';
 import { TransitionsService } from 'services/transitions';
 import { KeyListenerService } from 'services/key-listener';
 import { Inject } from 'util/injector';
-import { StatefulService, mutation, ServiceHelper } from 'services/stateful-service';
-import { defer } from 'lodash';
+import { mutation, ServiceHelper, StatefulService } from 'services/stateful-service';
 import { $t } from 'services/i18n';
+import * as obs from '../../obs-api';
 
-function getScenesService(): ScenesService {
-  return ScenesService.instance;
-}
+const getScenesService = (): ScenesService => ScenesService.instance;
 
-function getSourcesService(): SourcesService {
-  return SourcesService.instance;
-}
+const getSourcesService = (): SourcesService => SourcesService.instance;
 
-function getStreamingService(): StreamingService {
-  return StreamingService.instance;
-}
+const getStreamingService = (): StreamingService => StreamingService.instance;
 
-function getTransitionsService(): TransitionsService {
-  return TransitionsService.instance;
-}
+const getTransitionsService = (): TransitionsService => TransitionsService.instance;
 
-type THotkeyType = 'GENERAL' | 'SCENE' | 'SCENE_ITEM' | 'SOURCE';
+const isAudio = (sourceId: string) => {
+  const source = getSourcesService().getSource(sourceId);
```

Comment:
Is it really possible to get  `source = null` in this function?

Reply:
Yes, in my testing